portfolio/src/main/webapp/style.css

```diff
@@ -1,21 +1,27 @@
+/* Top margin for navbar and off-white background */
+body {
+    padding-top: 40px;
+}
+
+/* TODO: Make navbar responsive to small screens */
```

Comment:
Optional: Consider opening an issue for this on GitHub and referencing the issue here.

Reply:
I think that's a better idea, I've removed the TODO and will open an issue as a reminder to myself instead. Thanks!